### DISCUSSION TITLE
Bug with resource spawner overhaul or something else?

### DISCUSSION THREAD

**Discussion by ZeTBCN** [notnotmelon/factorissimo-2-notnotmelon#98] [General]
Ive had a game instance with mods including Factorismo 3 and it was working perfectly. Ive added couple of more mods such as Resource spawner overhaul and wanted to do a reactor in a factory building, noticed that it wasn't there. Checked, the mod was off. Turned it on and when inside the largest factory building all of the walls were gone. It was just another surface without any building limits. Great! But the transfer of resources between factory and outside doesn't work

**Comment by notnotmelon** [comment]
Yep, its fixed
